client-scripts-api.md

```diff
@@ -277,3 +277,109 @@ Payload:
   value: true|false
 }
 ```
+
+### `routeChange`
+
+This event is triggered when a change in the internal router happens.
+
+Payload:
+
+```js
+{
+  location: {
+    pathname: '/products',
+    search: '?page=0',
+    query: {
+      page: 0,
+    },
+    hash: '#',
+  },
+  action: 'PUSH',
+}
+```
+
+### E-Commerce specific events
+
+#### `ecomCheckout`
+
```

Comment:
the `ecom` prefix feels a bit superfluous:

```
checkout,
basketQuantityChange,
searchTermChange,
...
````
they're a bit more concise and can be used in more scenarios. For example searchTermChange can be used in google search studies

Reply:
I updated most event names to be more generic. For now I kept the `ecom` prefix for checkout and the basket quantity change. They are quite specific to ecom products. But that also is up to discussion.

I think it's generally good to have the possibility of context specific events and namespace them with a prefix.